src/main/java/wolox/training/models/User.java

```diff
@@ -98,6 +104,14 @@ public boolean removeBook(Book book) {
         return this.books.remove(book);
     }
 
+    public String getPassword() {
```

Comment:
This will serialize the encrypted password when getting the user. You should check JsonIgnore and JsonProperty annotations to control this and only let the setter work when deserializing, but not the getter when serializing.